green_bit_llm/inference/backends/green_bit_backend.py

```diff
@@ -0,0 +1,58 @@
+from green_bit_llm.inference.sim_gen import DTYPE
+from .base import BaseInferenceBackend
+import os
+
+import torch
+import torch.nn as nn
+
+import warnings
+warnings.filterwarnings("ignore", category=UserWarning, module='torch.nn.modules.module')
+
+from transformers import PreTrainedTokenizer
+
+from green_bit_llm.common import generate, load
+from green_bit_llm.args_parser import setup_shared_arg_parser
+
+# default value for arguments
+DEFAULT_PROMPT = None
+DEFAULT_MAX_TOKENS = 100
+DEFAULT_TEMP = 0.8
+DEFAULT_TOP_P = 0.95
+DTYPE = torch.half
+
+class GBLLMInferenceBackend(BaseInferenceBackend):
+    def __init__(self, model_path, **kwargs):
+        # Building configs
+        tokenizer_config = {"trust_remote_code": True if kwargs.get("trust_remote_code") else None}
```

Comment:
I think we should use `kwargs.get("trust_remote_code", None)` to set a default. Similar below for the same and other arguments.